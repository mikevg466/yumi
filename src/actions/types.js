export default {
  SET_NAME: 'SET_NAME',
  SET_GREETING: 'SET_GREETING',
  SELECT_RECIPE: 'SELECT_RECIPE',
};
