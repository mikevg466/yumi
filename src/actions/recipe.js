import types from './types';

export const selectRecipe = recipe => ({
  type: types.SELECT_RECIPE,
  recipe,
});
