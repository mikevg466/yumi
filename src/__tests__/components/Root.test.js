import React from 'react';
import { shallow } from 'enzyme';
import { bindActionCreators } from 'redux';
import RootComponent from '../../components/Root';
import { UserActions } from '../../actions';

const mockDispatchers = bindActionCreators(UserActions, jest.fn());

describe('Root Component', () => {
  it('renders', () => {
    const wrapper = shallow(
      <RootComponent
        actions={mockDispatchers}
        greeting='testGreeting'
        name='testName'
      />
    );
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
