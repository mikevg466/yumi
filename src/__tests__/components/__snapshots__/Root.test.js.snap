// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Component renders 1`] = `
"<div className=\\"Root\\">
  <Connect(RecipeContainer) />
</div>"
`;
